src/components/Header/PreviewAlert.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import cookie from 'js-cookie';
+import HeaderAlert from './HeaderAlert';
+import { useAtom } from 'jotai';
+import { isPreviewAtom } from '../../state/atoms/releaseAtom';
+import { isBeta } from '../../utils/common';
+import { AlertActionLink, AlertVariant } from '@patternfly/react-core/dist/dynamic/components/Alert';
+import { useLocation } from 'react-router-dom';
+import { useFlag } from '@unleash/proxy-client-react';
+
+const LOCAL_PREVIEW = localStorage.getItem('chrome:local-preview') === 'true';
+
+const PreviewAlert = ({ switchRelease }: { switchRelease: (isBeta: boolean, pathname: string, previewEnabled: boolean) => void }) => {
+  const [isPreview, togglePreview] = useAtom(isPreviewAtom);
+  const [prevPreviewValue, setPrevPreviewValue] = useState(isPreview);
+  const location = useLocation();
+  const previewEnabled = useFlag('platform.chrome.preview');
+
+  // FIXME: Remove the cookie check once the local preview is enabled by default
+  const shouldRenderAlert = LOCAL_PREVIEW ? isPreview !== prevPreviewValue : cookie.get('cs_toggledRelease') === 'true';
+  const isPreviewEnabled = LOCAL_PREVIEW ? isPreview : isBeta();
+
+  function handlePreviewToggle() {
+    if (!LOCAL_PREVIEW) {
+      switchRelease(isPreviewEnabled, location.pathname, previewEnabled);
+    }
+    togglePreview();
+  }
+
+  return shouldRenderAlert ? (
+    <HeaderAlert
```

Comment:
FWIW Github diff doesn't show - but this came from here originally: https://github.com/RedHatInsights/insights-chrome/pull/2835/files#diff-2e39c3da8c6be596802ae76544d42142fa129a8faf2e3406183c38b34b95dc73L383-L408